src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@ public interface MachineService {
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<T> getAll(Class<? extends Machine> type);
```

Comment:
The method `getAll` returns a `List<Machine>`, but the task suggests parameterizing the class with a specific wildcard bounding. Consider changing the return type to `List<T>` to reflect the generic type parameter `T extends Machine` used in the interface.